### DISCUSSION TITLE
Untranslated strings as translated on transifex

### DISCUSSION THREAD

**Discussion by shevernitskiy** [dfint/.github#4] [Q&A] [answered]
Hey, I found some amount of untranslated strings as translated. Is this normal thing?

<img width="862" alt="image" src="https://user-images.githubusercontent.com/28886342/206902100-133599f3-aa6e-40bb-bc0d-59151c3daa33.png">

**Comment by shevernitskiy** [comment]
Regex for at least one [a-z] word gives 46 matches. 

```regex
stringdump_steam_2022_12_06.txt:(?<number>[0-9]+)\n.+\nmsgstr\s".*(?<word>\s[a-z]+\s).*"
```


<img width="837" alt="image" src="https://user-images.githubusercontent.com/28886342/206903252-9c098c6f-2111-457f-8a1d-73f4b9a92e22.png">

**Reply by insolor** [reply, marked as answer]
This is not critical. As far as I know it appears in game logs (stdout.txt, gamelog.txt, stderr.txt), not in user interface.
